### DISCUSSION TITLE
Some of my thoughts on the Mechanic.

### DISCUSSION THREAD

**Discussion by Rotiart-Drol** [FakeFishGames/Barotrauma#11593] [Ideas and suggestions]
I'm not sure how balancing is going to be approached, if at all, in the post 1.0 environment. But if feedback is still being taken under advisement - particularly for talent balance - I would like to give some.

One of the things I noticed, compared to other classes such as security, is that Mechanic doesn't have quite as many talents that are as exciting or useful, or are perhaps give you things that don't compare favorably to other items already in the game. I'm not going to suggest any top-to-bottom reworks, but I think I have a few tweaks that could add some much appreciated utility and flavor for some talents.

Starting with the T6s, Iron Storm, Robotics, and MECHanic.

Iron Storm has the potential for being a DOA talent if your submarine crew is already using a T3 submarine, because the Scrap Cannon just doesn't compare very favorably to something like the harpoon gun, or harpoon coil, not necessarily because of the Scrap Cannon's raw stats, but rather because it's wasteful and cumbersome. If you shoot one steel bar out of a scrap cannon, it deals respectable damage, in a very unforgiving cone of fire, in exchange for your entire toolbelt slot and some battery charge. If you shoot a Harpoon Coil, it deals a little worse damage, but without a cone of fire to contend with, costing only 1/6 of a steel bar if you do not choose to retrieve your harpoon, in exchange for a hotbar slot and some battery charge. For most scenarios, the Coil Harpoon has the scrap cannon beat. However instead of putting it in stiffer competition with that weapon, I offer an alternative solution. One more flavorful to the Mechanic in specific.

Mechanics, being typically the class with the most downtime, are usually the ones that end up organizing the supplies of the submarine. This role is bolstered by the other talents in the Scrapper Tree. They would be uniquely aware of what material is more needed, and what the submarine doesn't need quite so much of. So I suggest that the Scrap Cannon be allowed to load, and fire _any_ metal, with the same statistics as firing raw iron out of the scrap cannon. Essentially this would be a way for the mechanic to organically keep the submarine organized without relying on outposts, or Honkmother forbid, dumping things out the airlock. In addition, I think the scrap cannon could benefit from being made into a medium item. I think these changes wouldn't make it super powerful, in fact it still probably wouldn't be anyone's first choice of weapon. But it would be flavorful, it would give some more agency to mechanics, and more things for them to plan around, which I think is exactly what a talent should do.

Robotics. I don't have as much to say on this as I did for Iron Storm. Defense Drones can actually output some good, accurate damage, it's just that I find they're almost never where you need them to be, and you as the mechanic are _highly_ reliant on either outposts or security officers to supply you with SMGs to replace your lost drones. Not to mention that they are very expensive for how fragile they are. The SMG recipe from the Defense Drone could be removed, and replaced it with it's constituent materials. That is to say, instead of the Defense Drone recipe calling for 1x FPGA, 1x SMG, and 2x Titanium-Aluminum Alloy, it would call for 1x FPGA, 1x Plastic, and 5x Titanium-Aluminum Alloy. I think the defense drones would benefit immensely from the ability to receive radio commands, like a bot would. Being able to order them to hold in a certain spot on the submarine, perhaps a known weakpoint, or to get them to follow you at a distance could make them worth the asking price. Again, I don't think the Defense Drones are going to rocket up the standings to dethrone any other combat T6, but they would be a little better, and a little more worth the asking price, which I think would make the game better over-all.

Mechanic's last T6 is MECHanic. Certainly the most exciting concept. But from my own anecdotal experience, and from [this poll](https://www.reddit.com/r/Barotrauma/comments/11brbwu/best_diving_suit/), notably from _before_ the 1.0 nerf, the Exosuit is pretty unpopular. Which is a shame because it, along with I am that guy is a tasteful Bioshock reference, and the exosuit itself is a nice looking sprite. If use-rate is important, the recent nerf changing the power requirement from batteries to fuel rods could be undone, but the thing is that nerf doesn't affect it in practical gameplay. Thorium is plentiful, a single quality thorium rod can fuel the suit for over 40 minutes, and precisely because the suit is fairly unpopular among players, the players that like the exosuit, aren't going to be forced to stop using the exosuit. Which I hope was the intention.

I'm satisfied with the exosuit, even if I feel the recent nerf wasn't justified. It's strong in it's niche against physical damage. It facilitates the blunt melee mechanic subclass into the late game, even if it isn't by any means an optimal style of play. 
It has it's weaknesses. Wearing it inside the submarine means that creatures are more likely to break into the sub, and while everything aside from a mudraptor veteran would be easily dispatched by an exosuit user, the fact remains that actually wearing the exosuit while you are supposed to be keeping the hull intact does more harm than good due to the hefty speed malus.
Furthermore, the speed malus means that any fight you get into, you are forced to commit to. So by turning a corner and being face to face with a steam guardian or two, or being ambushed by spinelings in a cave, the exosuit could get you killed, where if you had chosen a PUCS or a Slipsuit instead, you would have survived. This, to me, is balanced. The exosuit doesn't directly outshine _any_ other suit, but it brings some of it's own things to the table, which is why I'm satisfied with it, and think it could benefit from no major changes, unlike the other two T6s

Now, with the T6s out of the way, there's handful of other perks I would like to air some grievances about.
Engine Engineer, and Pump N Dump are very difficult for you, as the mechanic, to gauge it's effectiveness. It's very hard for you to understand what value, if any, you are receiving from it. This makes them less desirable than something like Iron Man, or Safety First, or Oiled Machinery. How that could be changed is beyond me, I mentioned them only because they contribute to the talent scarcity mechanic experiences that other classes do not. A security officer is bursting at the seams for T1s and T2s that have tangible impact on their characters/crew

Ballast Denizen. If it granted oxygen low resistance instead, it would be maybe the second or third best T1. It would be complimented well by Tiger Thresher genes, which I think isn't a very good gene as it stands. Buffing Ballast Denizen would be an opportunity to make both boats rise with the tide.

Modular Repairs gives you access to repair packs, which I view as somewhat wasteful. The crafting recipe could be reduced, or the output could be increased, perhaps a single craft could give you an entire stack, akin to how a single harpoon craft grants you an entire stack. This would make them a little more desirable. Yet the simple fact of the matter is I have never, while playing mechanic, thought to myself that I _need_ mechanical repairs to go a little faster. Mechanical items in need of repair are pretty few and far between, most of your time is spent on fixing hulls or junction boxes, and when you do need to repair a mechanical item, they're fast enough to repair, to the point that even a hefty increase in repair speed wouldn't translate to any impact on the round. Repair packs, I feel, are a convenience, so their cost of use should be adjusted accordingly.

Tinkerer. Perhaps I misunderstand how skill gain is calculated, but it seems to me that if you intend to get value out of this perk, your mechanical skill is overall going to increase slower, on the job that already has the slowest skill gain. In the late game you are completely reliant on wasteful craft spam or books to regain your lost skill, and it seems unfortunately that tinkerer only exacerbates the issue. If tinkerer didn't do that, while still not very interesting it would at least be a more formidable opponent to Quickfixer

Lastly, and speaking of, I feel like Quickfixer's speed boost is mostly a dopamine hit to the user. Getting somewhere merely 20% faster on land only I feel is very insignificant. I think Quickfixer could benefit from the buff being longer, something like 30 seconds, or even up to 60 seconds, and self-stacking would give it a more tangible impact, and make it an exciting pick. Something that stands out even among talents from other classes. Given how relatively uncommon mechanical breakdowns are on the submarine, I still feel like this wouldn't have a big impact on every round. But if you get hit by an EMP or are fighting a Black Moloch... Well, it would certainly stand equal to it's piers from the other trees, like Mass Production or Mudraptor Wrestler

**Comment by NoIWontBruh0831** [comment]
I dont know if replying to a thread increases its visibility or not, but I completely agree with everything you wrote and would like this post to be more popular lol

**Comment by BiffW** [comment]
About "Robotics".
I think it can be more fun to replace the gun on the drone with a pair of hands.
Let the players to decide which weapon it should use. And it provide more possibilities for mods to make suitable weapons for the defense drone.
And I do agree that defense drone is too fragile compared to materials required to fabricate it.
